### PR TITLE
Add metrics_data to Zipcode

### PR DESCRIPTION
**Pull request by e3klund**

### CODE REVIEW

**Comment by ianhoffman** [comment]
I think you may need to run `black .` again now that you merged with master @e3klund

File: app/airq/models/zipcodes.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ class Zipcode(db.Model):  # type: ignore
     pm25_updated_at = db.Column(
         db.Integer(), nullable=False, index=True, server_default="0"
     )
+    # JSON BLOB
```
Comment (ianhoffman) [inline app/airq/models/zipcodes.py:47]: can we delete this comment?

Comment (e3klund) [reply app/airq/models/zipcodes.py:47]: yes will do

**Comment by e3klund** [comment]
@ianhoffman ready for your review

**Review by ianhoffman — APPROVED** [review]
Lgtm

**Pull request merged**